.github/workflows/ci.yml

```diff
@@ -54,9 +57,8 @@ jobs:
       - name: Build
         run: |
           # Download nydus components
-          NYDUS_VER=v$(curl -s "https://api.github.com/repos/dragonflyoss/image-service/releases/latest" | jq -r .tag_name | sed 's/^v//')
-          wget https://github.com/dragonflyoss/image-service/releases/download/$NYDUS_VER/nydus-static-$NYDUS_VER-linux-amd64.tgz
-          tar xzvf nydus-static-$NYDUS_VER-linux-amd64.tgz
+          wget https://github.com/dragonflyoss/image-service/releases/download/${{ env.NYDUS_VERSION }}/nydus-static-${{ env.NYDUS_VERSION }}-linux-amd64.tgz
```

Comment:
Merge confliction? In the main branch, we have used the latest release to run the CI.

See https://github.com/containerd/nydus-snapshotter/blob/0c40977cc4898ee0f043dbb00390dedb35518803/.github/workflows/ci.yml#L57